### PR TITLE
ENGG-2054: Session viewer bottom sheet panel

### PR DESCRIPTION
**Pull request by RuntimeTerror10**
Closes issue: 

## 📜 Summary of changes:
This PR adds bottom sheet panel in session viewer

Screenshot 2024-07-17 at 9 13 55 AM

## ✅ Checklist:

- [ ] Make sure linting and unit tests pass.
- [ ] No install/build warnings introduced.
- [ ] Verified UI in browser.
- [ ] For UI changes, added/updated analytics events (if applicable).
- [ ] For changes in extension's code, manually tested in Chrome and Firefox.
- [ ] For changes in extension's code, both MV2 and MV3 are covered.
- [ ] Added/updated unit tests for this change.
- [ ] Raised pull request to update corresponding documentation (if already exists).

### CODE REVIEW

**Review by rohanmathur91 — COMMENTED** [review]
Enhancement: Both panels should be expandable I feel via dragging in between, you can log a ticket for this for future.

Screenshot 2024-07-19 at 12 53 48 PM

**Review by rohanmathur91 — COMMENTED** [review]
Is this how the overflow is expected? 
Screenshot 2024-07-19 at 12 51 55 PM

**Review by rohanmathur91 — COMMENTED** [review]
Possible to increase the height of the video panel when bottomsheet is on left?
Screenshot 2024-07-19 at 12 55 22 PM

File: app/src/features/sessionBook/screens/SavedSessionScreen/components/SessionViewerBottomSheet/components/SessionInfo/SessionInfo.tsx [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+  const sessionAttributes = useSelector(getSessionRecordingAttributes);
+
+  const handleSessionNameUpdate = useCallback(() => {
+    if (recordingId && sessionMetadata?.name) {
+      updateSessionName(user?.details?.profile?.uid, recordingId, sessionMetadata.name);
+    }
+  }, [recordingId, sessionMetadata?.name, user?.details?.profile?.uid]);
+
+  const handleSessionDescriptionUpdate = useCallback(() => {
+    if (recordingId && sessionMetadata?.description) {
```
Comment (rohanmathur91) [inline app/src/features/sessionBook/screens/SavedSessionScreen/components/SessionViewerBottomSheet/components/SessionInfo/SessionInfo.tsx:34]: Confirm this if we need analytics over here or not.

File: app/src/features/sessionBook/screens/SavedSessionScreen/components/SavedSessionViewer/SavedSessionViewer.tsx [hunk 2]
```diff
@@ -72,8 +72,7 @@ export const SavedSessionViewer = () => {
             <DownloadSessionButton />
           </div>
         </div>
-        {/* TODO: Add bottom sheet */}
-        <BottomSheetLayout bottomSheet={<>Session Bottom sheet here</>}>
+        <BottomSheetLayout bottomSheet={<SessionViewerBottomSheet playerTimeOffset={sessionPlayerOffset} />}>
```
Comment (rohanmathur91) [inline app/src/features/sessionBook/screens/SavedSessionScreen/components/SavedSessionViewer/SavedSessionViewer.tsx:75]: Are we sending any analytics event for bottomsheet?

File: app/src/features/sessionBook/screens/SavedSessionScreen/components/SessionViewerBottomSheet/components/SessionInfo/SessionInfo.tsx (hunk 1 above)
Comment (RuntimeTerror10) [reply app/src/features/sessionBook/screens/SavedSessionScreen/components/SessionViewerBottomSheet/components/SessionInfo/SessionInfo.tsx:34]: An event is being already sent inside `updateSessionName` funtion

File: app/src/features/sessionBook/screens/SavedSessionScreen/components/SavedSessionViewer/SavedSessionViewer.tsx (hunk 2 above)
Comment (RuntimeTerror10) [reply app/src/features/sessionBook/screens/SavedSessionScreen/components/SavedSessionViewer/SavedSessionViewer.tsx:75]: BottomSheet itself has some analytics events, and I've also added tab clicked events in this PR: 
https://github.com/requestly/requestly/pull/1969

**Review by rohanmathur91 — APPROVED** [review]

**Pull request merged**